connectors/sources/generic_database.py

```diff
@@ -196,7 +198,7 @@ async def validate_config(self):
             raise Exception("Configured port has to be an integer.")
 
         if (
-            self.dialect == "Postgresql"
+            self.dialect != "Oracle"
```

Comment:
Where is this change coming from?

Reply:
because in current main branch we have removed `validate_config` method so it's not present in main but we need to keep this in 8.7. I have added this change in original PR but after taking pull from main branch it was removed from the original PR.